static/api.yaml

```diff
@@ -0,0 +1,386 @@
+openapi: '3.0.0'
+info:
+  title: Decentraland Governance API
+  version: '1.0.0'
+servers:
+  - url: localhost:4000/api
+tags:
+  - name: Proposals
+    description: Everything about DAO proposal
+  - name: Coauthor
+    description: Everything about proposal coauthors
+  - name: Updates
+    description: Everything about grant updates
+  - name: Committee
+    description: View DAO Committee members
+paths:
```

Comment:
We are missing this ones from budgets that I think are relevant:

'/budget/current'
'/budget/contested/:proposal'
'/budget/:category'

But I'm ok to merge and add them later